doc/releases/migration-guide-3.5.rst

```diff
@@ -0,0 +1,36 @@
+:orphan:
+
+.. _migration_3.5:
+
+Migration guide to Zephyr 3.5.0 (Working Draft)
+###############################################
+
+This document describe the changes required or recommended when migrating your
+application to Zephyr 3.5.0.
+
+Required changes
+****************
+
+* The :kconfig:option:`CONFIG_BOOTLOADER_SRAM_SIZE` default value is now ``0`` (was
```

Comment:
as the list grows, it will very helpful and useful to add sections here based on the areas affected. So, bootloader, gpio, mcumgr, kernel, etc...

Reply:
I thought about it, but for now I think we can keep it simple.